bin/check-consul-servers.rb

```diff
@@ -65,8 +65,23 @@ class ConsulStatus < Sensu::Plugin::Check::CLI
          long: '--scheme SCHEME',
          default: 'http'
 
+  option :insecure,
+         description: 'if set, disables SSL verification',
+         short: '-k',
+         long: '--insecure'
+
+  option :capath,
+         description: 'absolute path to an alternate CA file',
+         short: '-c CAPATH',
+         long: '--capath CAPATH'
+
   def run
-    json = RestClient::Resource.new("#{config[:scheme]}://#{config[:server]}:#{config[:port]}/v1/status/peers", timeout: 5).get
+    url = "#{config[:scheme]}://#{config[:server]}:#{config[:port]}/v1/status/peers"
+    options = { timeout: 5,
```

Comment:
I think we should expose the timeout as an option, someone will inevitably say that it's reasonable to wait more than 5 seconds.